src/components/search/SearchPage.jsx

```diff
@@ -8,29 +8,42 @@ import Fade from "react-reveal/Fade";
 import queryString from "query-string";
 import NavigationBar from "../NavigationBar";
 import SearchItem from "./SearchItem";
+import * as friendsService from "../../services/FriendService";
 
 class SearchPage extends Component {
   constructor(props) {
     super(props);
     this.props = props;
     this.state = {
-      // Todo: query on api
       resultsList: [],
       keyword: queryString.parse(props.location.search).username,
+      loading: true,
     };
   }
 
+  componentDidMount() {
+    const { keyword } = this.state;
+    friendsService.searchAuthors(keyword).then((response) => {
+      this.setState({
+        resultsList: response,
+        loading: false,
+      });
+    }).catch((error) => {
+      // eslint-disable-next-line no-alert
+      alert(error);
+    });
+  }
+
   renderSearchResults = () => {
     const results = [];
     const { resultsList } = this.state;
-
+    const { user } = this.props;
```

Comment:
I think you forgot to change from `user` to `currentUser` in this file. There are 5 lines you need to update.

Reply:
fixed